docs/source/installation.rst

```diff
@@ -100,98 +100,96 @@ Step 1: Setup dedicated python environment
             If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_.
 
 
-Step 2: Install ``ivadomed``
-++++++++++++++++++++++++++++
-
+Step 2: Install ``ivadomed`` with GPU or CPU Support
+++++++++++++++++++++++++++++++++++++++++++++++++++++
 
     .. tabs::
 
-        .. group-tab:: PyPI Installation
-
-            Install ``ivadomed`` and its requirements from
-            `PyPI <https://pypi.org/project/ivadomed/>`__:
-
-            ::
-
-                pip install --upgrade pip
+        .. tab:: NVIDIA GPU Support
 
-                pip install ivadomed
+            ``ivadomed`` requires CUDA 11.1 to execute properly. If you have an NVIDIA GPU, look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`__, which needs to be > 3.5 to be compatible with CUDA 11.1. Also, make sure to upgrade your NVIDIA driver to at least the minimum required version as indicated `here <https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility>`__.
 
-        .. group-tab:: Repo Installation (Advanced or Developer)
+            To verify the CUDA version, simply run ``nvcc -V`` in both Linux and Windows. The release refers to your current CUDA version.
 
-            Bleeding-edge developments are available on the project's master branch
-            on Github. Install ``ivadomed`` from source:
-
-            ::
+            .. tabs::
 
-                git clone https://github.com/ivadomed/ivadomed.git
+                .. tab:: Package Installation (Recommended)
 
-                cd ivadomed
+                    After verifying CUDA 11.1 installation, install ``ivadomed`` from `PyPI <https://pypi.org/project/ivadomed/>`__:
+                    
+                    ::
 
-                pip install -e .
+                        pip install --upgrade pip
 
+                        pip install ivadomed[gpu]
 
-Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    .. tabs::
+                .. tab:: Source Installation
```

Comment:
Could try ` .. group-tab:: Source Installation` and allow sync between the GPU and CPU page when switching.